indico/modules/categories/models/categories.py

```diff
@@ -367,7 +367,7 @@ def _mappers_configured():
     query = (db.select([db.func.count(Event.id)])
              .where((Event.category_id == Category.id) & ~Event.is_deleted)
              .correlate_except(Event))
-    Category.event_count = column_property(query, deferred=True)
+    Category.event_count = column_property(query)
```

Comment:
Uh, no. The point of having it deferred by default is to avoid querying information that is almost never needed. Use `.options(undefer('event_count'))` on the query instead.